Bottom: bbbd549460a7ee70f135aba05a5c27a05e851413
Top:    5073130fbc3b00c38b59e71b55dfcc956c3e454f
Author: Jon Smirl <jonsmirl@terra>
Date:   2012-06-09 16:20:04 -0400

Make sure mmc stop command is fully initialized before issuing the parent command


---

diff --git a/drivers/mmc/host/lpc31xx_mmc.c b/drivers/mmc/host/lpc31xx_mmc.c
index f04d4c7..31b8f78 100644
--- a/drivers/mmc/host/lpc31xx_mmc.c
+++ b/drivers/mmc/host/lpc31xx_mmc.c
@@ -611,11 +611,10 @@ static void lpc313x_mci_start_request(struct lpc313x_mci *host,
 	if (data) //we may need to move this code to mci_start_command
 		lpc313x_mci_submit_data(host, data);
 
-	lpc313x_mci_start_command(host, cmd, cmdflags);
-
 	if (mrq->stop) 
 		host->stop_cmdr = lpc313x_mci_prepare_command(slot->mmc, mrq->stop);
-	
+
+	lpc313x_mci_start_command(host, cmd, cmdflags);
 }
